frontend_lib/src/mentionOrLink.js

```diff
@@ -40,6 +41,7 @@ const wrapMentionsFromText = (text, doc, invalidMentionList) => {
   if (invalidMentionList.indexOf(match[0]) === -1) {
     const wrappedMention = doc.createElement(MENTION_TAG_NAME)
     wrappedMention.className = MENTION_CLASS
+    // wrappedMention.dir = 'ltr'
```

Comment:
Note: this line will be deleted or uncommented according to the answer of the Arabic expert.